### ISSUE TITLE
List scrolls when selecting note

### ISSUE THREAD

**Issue by claell** [Laverna/laverna#309]
When selecting a note in a scrollable category (I chose to have 20 notes displayed per page) the list is automatically scrolled so the note fits a certain position. This behaviour is good for notes at the end of the viewable field of the list, but not for ones in the middle. I hope I could describe the problem, else just ask.

**Comment by wwebfor** [comment]
Is it a bug or an unwanted behaviour?

**Comment by claell** [comment]
I think an unwanted behaviour, don't know. When selecting the note in the middle it gets scrolled until it is at the top for example.

**Comment by wwebfor** [comment]
We implemented this behaviour in order to make an active note always visible in notes list. Is it working in a wrong way?

**Comment by claell** [comment]
I understand. But when selecting a note in the middle of the screen, why is there a need to move it to the top?

**Comment by wwebfor** [comment]
Now I understood what you mean. Only when a note is the very last visible one on the screen, it should scroll. Did I understand correctly? In that case I totally agree, it is indeed a more intelligent way of dealing with this problem.

**Comment by claell** [comment]
Yes this is what I mean (additional I would also use this functionality when there is a note on top half visible and maybe scroll the last visible note on the bottom just sligtly up and not all to the top).
